docs/index.md

```diff
@@ -36,34 +43,45 @@ How-to guides provide step-by-step instructions for a wide range of tasks:
 - [Representing experimental geometries](how-tos/transformations.md)
 - [Using pynxtools test framework](how-tos/using-pynxtools-test-framework.md)
 
-
 </div>
 
 <div markdown="block">
 
 ### Learn
 
-An introduction to NeXus and its design principles.
+#### An introduction to NeXus and its design principles
 
 - [An introduction to NeXus](learn/nexus-primer.md)
 - [Rules for storing data in NeXus](learn/nexus-rules.md)
 - [The concept of multiple application definitions](learn/multiple-appdefs.md)
+
+#### pynxtools
+
+- [Data conversion in `pynxtools`](learn/dataconverter-and-readers.md)
+- [NeXus verification in `pynxtools`](learn/nexus-verification.md)
 - [The MultiFormatReader as a reader superclass](learn/multi-format-reader.md)
 
 </div>
 <div markdown="block">
 
 ### Reference
 
+#### NeXus definitions
+[Here](reference/definitions.md), you find the detailed list of application definitions and base classes and their respective fields.
+
+Or go directly to the [official NIAC](https://manual.nexusformat.org/classes/index.html)
+ or [latest FAIRmat](https://fairmat-nfdi.github.io/nexus_definitions/) definitions.
+
+#### pynxtools
+
 `pynxtools` has a number of command line tools that can be used to convert data and verify NeXus files. You can more information about the
 API [here](reference/cli-api.md).
 
-Within FAIRmat, we maintain a number of reader plugins for different experimental techniques. You can find more information [here](reference/plugins.md).
+Within FAIRmat, we maintain a number of pynxtools readers as well as reader plugins for different experimental techniques. Here you can find more information:
 
-[Here](reference/definitions.md), you find the detailed list of application definitions and base classes and their respective fields.
+- [Built-in pynxtools readers](reference/built-in-readers.md)
+- [FAIRMat-suppored pynxtools plugins](reference/plugins.md)
```

Comment:
I call them reader here because in the pynxtools framework these things are called readers. We have the NexusParser, but this _calls_ the internal readers.